### PR TITLE
repo: Change to dora-metrics organization

### LINKED ISSUE #887: Relocate the Pelorus github repo outside of konveyor
CNCF is requesting that Pelorus be moved out of the konveyor repo.

* best location atm is https://github.com/migtools/  unless we want our own org.

- [ ] https://github.com/dora-metrics/pelorus/pull/896
  - [x] test operator with `operator-sdk`
  - [x] dpulls app
  - [x] [dco app](https://github.com/apps/dco)
- [x] https://github.com/dora-metrics/pelorus/pull/910
- [x] https://github.com/konveyor/mig-demo-apps/pull/97
- [x] https://github.com/openshift/release/pull/37757
  - [x] https://docs.ci.openshift.org/docs/how-tos/onboarding-a-new-component/

### PR DESCRIPTION
**Pull request by mateusoliveira43**
## Describe the behavior changes introduced in this PR

Move Pelorus project to https://github.com/dora-metrics organization.

## Linked Issues

resolves #887

## Testing Instructions

I have run `grep -Iinr "konveyor" . --exclude-dir={.venv,pelorus-operator,.pytest_cache}` to help me change `konveyor` to `dora-metrics`. 

We still need to

- [x] Create CODE_OF_CONDUCT.md in `dora-metrics` org. Depends on: https://github.com/dora-metrics/pelorus/pull/910
- [x] Test if nothing breaks
  - [x] Test Operator `operator-sdk run bundle quay.io/msouzaol/pelorus-operator-bundle:v0.0.11 --namespace pelorus` 
- [x] Check if run-pelorus-e2e-tests needs changes (and makefile commands)
  - [x] Change https://github.com/konveyor/mig-demo-apps `apps/todolist-mongo-go/pelorus` files

Did not change pelorus operator folder, as I think is best to first have Michal's PR there, but I have run the create script on this branch and only got this with `grep`
```
./pelorus-operator/bundle/manifests/pelorus-operator.clusterserviceversion.yaml:143:    [YouTube Video](https://www.youtube.com/watch?v=VPCOIfDcgso) with the Pelorus in action recorded during online Konveyor Community event.
```

### CODE REVIEW

**Review by mpryc — APPROVED** [review]

File: scripts/pelorus-operator-patches/04_pelorus_cluster_service_versions.diff [hunk 1]
```diff
@@ -106,18 +106,18 @@
...
++    If you are interested in contributing to the Pelorus project, please review our Contribution guide which can be found in the [contribution guide](https://github.com/dora-metrics/pelorus/blob/master/CONTRIBUTING.md)
 +
 +    ## Statement of Support
 +
-+    Our support policy can be found in the [Upstream Support statement](https://github.com/konveyor/pelorus/blob/master/docs/UpstreamSupport.md)
++    Our support policy can be found in the [Upstream Support statement](https://github.com/dora-metrics/pelorus/blob/master/docs/UpstreamSupport.md)
 +
 +    ## Code of Conduct
-+    Refer to Konveyor's Code of Conduct [here](https://github.com/konveyor/community/blob/main/CODE_OF_CONDUCT.md).
++    Refer to dora-metrics's Code of Conduct [here](https://github.com/dora-metrics/community/blob/main/CODE_OF_CONDUCT.md).
```
Comment (mateusoliveira43) [inline scripts/pelorus-operator-patches/04_pelorus_cluster_service_versions.diff:116]: need to change?

Comment (mpryc) [reply scripts/pelorus-operator-patches/04_pelorus_cluster_service_versions.diff:116]: https://github.com/dora-metrics/pelorus/pull/910

**Comment by mpryc** [comment]
/hold

We need to prepare new Operator release just after this change. Otherwise we will advice to use non-existing CSV.

**Comment by mpryc** [comment]
/unhold

There is acceptance that doc will be not in sync until we release 0.0.6 of operator, but to make 0.0.6 we need this change.

**Review by mpryc — APPROVED** [review]

**Pull request merged**